src/plugins/info/plugin.py

```diff
@@ -103,6 +110,57 @@ def on_config(self, config):
             log.error("Please remove 'hooks' setting.")
             self._help_on_customizations_and_exit()
 
+        # Assure that possible relative paths, which will be validated
```

Comment:
Please make sure that comments always span to 80 characters. We use 80 characters everywhere.